services/web-app/pages/assets/guidelines/icons/contribute.mdx

```diff
@@ -0,0 +1,120 @@
+consistency and proper formatting.
+
+- All icons should be unique and not redundant with any existing icons in the system. Search the
+  [library](/guidelines/icons/library) for the keyword(s) associated with your proposed new icon to
+  ensure that it is not already represented.
+- All icons should adhere to the
+  [IBM Design Language visual style](https://www.ibm.com/design/language/iconography/ui-icons/design).
+- All icons should comply with IBM [accessibility standards](/guidelines/accessibility/overview).
+- All icons should be usable across all supported platforms and devices.
+- All icons should be understandable by a global audience of users, regardless of nationality or
+  language.
+
+## Making an icon
+
+- Create a 32px x 32px artboard for each icon.
+- 32 px icons should have 2 px padding.
+- Set your workspace up from the start to snap to pixels and round values to whole pixels to avoid
+  correcting shapes later.
+- Never use center borders. Centering can cause half pixels.
+- Avoid using the line tool; use the rectangle tool instead. The line tool will place the vector on
+  half pixels.
+- Be aware of automatic alignments which can place vectors on uneven or half pixels.
+- Ungroup icon layers completely. The icon should be on the top-most layer in your artboard.
+- Make sure to properly name layers and artboards _(these names will also be exported into the
+  code)_.
+- Review the
+  [icon master file](https://github.com/carbon-design-system/carbon/tree/v10/packages/icons/master)
+  to see these guidelines in practice.
+
+### Production-ready
+
+To be considered production-ready, all icon submissions must be delivered in SVG format at 32px x
+32px for integration with Carbon components.
+
+- Icons should be at whole pixels. No decimals are allowed in x and y coordinates or width and
+  height fields.
+- Each artboard and the artwork within it must be aligned to the pixel grid.
+- All strokes must be expanded and at full pixel values.
+- All possible shapes and paths should be combined.
+
+## Exporting SVGs
+
+### Export SVGs from Adobe Illustrator.
+
+1. Draw a 32px x 32px artboard. 32px icons should have 2px padding.
+2. Place the icon squarely on the artboard, making sure it's aligned to the pixel grid.
+3. Expand all strokes `Object` → `Expand`.
+4. Select all overlapping shapes and click the "Unite" icon in the Pathfinder panel to merge all of
+   the shapes together.
+5. Make sure the icon is at `#000000` and has no additional styling.
+6. Select `File` → `Save a Copy...` from the top navigation.
+7. On the `Format` dropdown select "SVG".
+8. Below `Format` select `Use Artboard`, then select either all or a range of artboards, depending
+   on your need.
+9. Click `Save`.
+10. The `SVG Options` dialog will then open.
+11. Make sure the preferences are the same as in the image below.
+
+![export an icon from illustrator](/guidelines/icons/iconography-exportsvgfromai.png)
```

Comment:
this image is rendering a lot bigger on platform 🤔 
<img width="1302" alt="image" src="https://user-images.githubusercontent.com/40550942/167732195-2413d551-2f82-4e4f-82c5-2ed191cdd951.png">
<img width="683" alt="image" src="https://user-images.githubusercontent.com/40550942/167732219-b1ff36a1-ae9a-4be8-a4ae-66fd755762af.png">

Reply:
Ah yea, this was one of the notes I made when migrating. Any images we want to display at their exact width/height instead of 100% width we will have to specifically set that. Or update the way we render images. I'll fix this.